webpack/basicModules.config.js

```diff
@@ -39,6 +39,8 @@ module.exports = ( mode ) => ( {
 		// Analytics advanced tracking script to be injected in the frontend.
 		'analytics-advanced-tracking':
 			'./assets/js/analytics-advanced-tracking.js',
+		// Consent Mode script.
+		'consent-mode': './assets/js/consent-mode/consent-mode.js',
```

Comment:
This isn't really the right group for the script since "basic modules" are essentially unoptimized.

The intention of this issue is to extract the source to its own file/module to allow for tooling, testing, and the use of modern language features and leave the transpilation to widely-supported JS syntax to Babel. Since we don't have any such modules in the plugin yet, we likely need to create a new Webpack config for generating such output as the Babel config for SK entrypoints in wp-admin won't be as broad in terms of compatibility we want to have on the frontend.

Reply:
Thanks @aaemnnosttv, yes that's right. [I've corrected this](https://github.com/google/site-kit-wp/pull/8893/commits/16d175dd1b1cb2279a8a06e75fbe0c28093a05b6)

Reply:
In the end I renamed the "Event Providers" build config to be "Frontend Scripts" instead and updated the webpack config to use `@babel/preset-env` to transpile the scripts to support any browser which is not dead and has over 0.25% worldwide market share according to browserlist in the commit f9ea3e3ea2cac0d8a24acc2e0c21df84bbbb5cfa

Reply:
This entry is a duplicate that still needs to be removed.